homeassistant/components/homekit/type_lights.py

```diff
@@ -55,8 +55,8 @@
 
 CHANGE_COALESCE_TIME_WINDOW = 0.01
 
-DEFAULT_MIN_MIREDS = 153
-DEFAULT_MAX_MIREDS = 500
+DEFAULT_MIN_COLOR_TEMP = 2000  # 153 mireds
+DEFAULT_MAX_COLOR_TEMP = 6535  # 500 mireds
```

Comment:
I think this probably safer to use 6500 since most devices have that as an upper limit and the difference is likely the conversion loss